src/mcu.cpp

```diff
@@ -1350,7 +1396,13 @@ int main(int argc, char *argv[])
             else if (!strcmp(argv[i], "-gm"))
             {
                 resetType = ResetType::GM_RESET;
-            }
+            }           
+        }
+
+        if (audioFormat == AudioFormat::FLOAT32 && !isBufferSet)
+        {
```

Comment:
In that case the argument order would be mandatory (that is -af has to be set after -ab) otherwise at 
(!strcmp(argv[i], "-af")) you cannot know if -ab is not set at all or only will be set after -af.
The if (audioFormat == AudioFormat::FLOAT32 && !isBufferSet) check is added to make sure that only change the dafault if -ab is not set manually.